Makefile

```diff
@@ -88,12 +93,20 @@ test/ibm-validation:
 	@echo Running the operator image in the cluster
 	operator-sdk test local ./test/e2e --go-test-flags "-tags=integration -coverpkg ./... -coverprofile cover-e2e.coverprofile -covermode=count -timeout 0" --operator-namespace $(NAMESPACE) --debug --verbose --global-manifest=deploy/empty-init.yaml --namespaced-manifest=deploy/operator.yaml
 
-.PHONY: test/e2e-local-image cluster/prepare setup/operator-sdk
-test/e2e-local-image: cluster/prepare setup/operator-sdk
-	@echo Running e2e tests with a fresh built operator image in the cluster:
+.PHONY: test/e2e-local-image setup/operator-sdk
+test/e2e-local-image: setup/operator-sdk
+	@echo Backing up operator.yaml
+	@cp deploy/operator.yaml deploy/operator.yaml_bckp
+	@echo Building operator image:
+	eval $$(minikube -p minikube docker-env); \
 	docker build . -t keycloak-operator:test
-	@echo Running tests:
-	operator-sdk test local --go-test-flags "-tags=integration -coverpkg ./... -coverprofile cover-e2e.coverprofile -covermode=count -timeout 0" --image="keycloak-operator:test" --operator-namespace $(NAMESPACE) --up-local --debug --verbose ./test/e2e
+	@echo Modifying operator.yaml
+	@sed -i 's/imagePullPolicy: Always/imagePullPolicy: Never/g' deploy/operator.yaml
```

Comment:
Is this necessary? I guess it will work even without it, right?

Reply:
Yes, it wouldn't work without it. It would try to pull the image from quay thanks to:
https://github.com/keycloak/keycloak-operator/blob/20a8954e07901cc731d53a6b42bb17d82e69c152/deploy/operator.yaml#L22

Reply:
Ok, thanks!